plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -118,6 +124,7 @@ func newQuotaEvaluator(client clientset.Interface, registry quota.Registry) (*qu
 		registry:        registry,
 		liveLookupCache: liveLookupCache,
 		liveTTL:         time.Duration(30 * time.Second),
+		updatedQuotas:   map[string]*api.ResourceQuota{},
```

Comment:
I guess the issue here is that if a namespace was deleted, or if the quota itself was deleted, this map would never remove that entry.  Is there anyway we can have the lookaside cache cleanup happen elsewhere?